package.json

```diff
@@ -95,6 +95,7 @@
         "@unibeautify/beautifier-prettydiff": "^0.6.0",
         "@unibeautify/beautifier-sqlformat": "^0.0.1",
         "cosmiconfig": "^5.0.6",
+        "csscomb": "^4.2.0",
```

Comment:
Error in Travis CI build:

```
src/beautifiers.ts:10:21 - error TS2307: Cannot find module '@unibeautify/beautifier-csscomb'.
```

Looks like `@unibeautify/beautifier-csscomb` is missing. Need to add it as dependency. 🎉